File: Training/PYTHON/Penilaian.py
print("====================================")
print("= PENILAIAN MAHASISWA ILKOM UNIMED =")
print("====================================")

absen = float(input("Masukkan Nilai Absen = "))
uts = float(input("Masukkan Nilai UTS = "))
uas = float(input("Masukkan Nilai UAS = "))
tugasKKNI = float(input("Masukkan Nilai Tugas KKNI = "))

totalNilai = 0.2 * absen + 0.2 * uts + 0.25 * uas + 0.5 * tugasKKNI

print("=======================")
print("INDEX MAHASISWA ADALAH")

if totalNilai >= 90:
            print("A = Sangat Kompeten")
elif totalNilai >= 80 and totalNilai < 89:
            print("B = Kompeten")
elif totalNilai >= 70 and totalNilai < 79:
        print("C =  Cukup Kompeten")
else:
        print("E = Tidak Kompeten")

File: Training/Penilaian.java
import java.util.Scanner;
public class Penilaian {
            public static void main(String[] args) {
                        Scanner input = new Scanner(System.in);

                        Float absen, tugasKKNI, uts, uas, totalNilai;

                        System.out.println("====================================");
                        System.out.println("= PENILAIAN MAHASISWA ILKOM UNIMED =");
                        System.out.println("====================================");
            
                        System.out.print("Masukkan Nilai Absen = ");
                        absen =  input.nextFloat();
                        System.out.print("Masukkan Nilai Tugas KKNI = ");
                        tugasKKNI =  input.nextFloat();
                        System.out.print("Masukkan Nilai UTS = ");
                        uts =  input.nextFloat();
                        System.out.print("Masukkan Nilai UAS = ");
                        uas =  input.nextFloat();

                        totalNilai = 0.05f * absen + 0.2f * uts + 0.25f * uas + 0.5f * tugasKKNI;

                        System.out.println("====================================");
                        System.out.print("INDEX MAHASISWA ADALAH ");

                        if (totalNilai >= 90) {
                                    System.out.println("A = Sangat Baik");        
                        } else if (totalNilai >= 80 && totalNilai < 89) {
                                    System.out.println("B = Baik");        
                        } else if (totalNilai >= 70 && totalNilai < 79) {
                                    System.out.println("C = Cukup");        
                        }  else{
                                    System.out.println("E = Sangat Kurang");
                     }

                     input.close();
            }

}
